pkg/manager/task_scan.go

```diff
@@ -47,9 +47,14 @@ func (j *ScanJob) Execute(ctx context.Context, progress *job.Progress) {
 
 	logger.Infof("Scan started with %d parallel tasks", parallelTasks)
 
+	var minModTime time.Time
+	if input.MinModTime != nil {
+		minModTime = *input.MinModTime
+	}
+
```

Comment:
Does this need to be here?
Can't it be inside the queueFiles function?
That way we can get the input directly eg `j.input.MinModTime` and not use an extra argument in the queueFiles function

Reply:
You are right, it needs to be pushed down.